packages/core/src/BacktraceCoreClient.ts

```diff
@@ -0,0 +1,8 @@
+import { BacktraceAttachment } from './model/report/BacktraceAttachment';
+import { BacktraceReport } from './model/report/BacktraceReport';
+
+export class BacktraceCoreClient {
+    public async send(report: BacktraceReport, attributes: Record<string, any>, attachments: BacktraceAttachment) {
```

Comment:
why `attributes` and `attachments` are not part of `BacktraceReport`?

Reply:
Done. Thx